docs/adr/0002-data-exploration.md

```diff
@@ -0,0 +1,52 @@
+# Provide options for further data exploration
+
+- Status: proposed
+- Deciders: Rob Miller, Rob Hudson, Hamilton Ulmer
+- Date: 2020-10-07
+
+## Context and Problem Statement
+
+GLAM is designed to answer a general set of questions against a limited set of
+dimensions. Sometimes the underlying data can tell us more than GLAM allows. For
+those users who want to dig deeper into the data, GLAM should provide jump off
+points to do so.
+
+## Decision Drivers
+
+- What is best to provide to end users to get their questions answered?
+- What can we accomplish with our current resources for the generalized case,
+  but also help enable non-generalized use cases?
+
+## Considered Options
+
+Currently we provide the JSON representation of the data being displayed as an
+"Export to JSON" link
+
+Extra options we can provide:
+
+1. We could provide other alternate data file formats (e.g. CSV)
+2. We could provide the SQL to query the current set of data being displayed to
+   the user which can then be copy/pasted into Redash or BQ console as a
+   starting point for further exploration
+3. We could link directly to Redash with the query pre-populated
```

Comment:
Option 3 is pretty much a non-starter... given the BI tool evaluation that is currently happening, we don't know what role Redash will serve going forward, investing in a tighter integration with Redash would be unwise right now.